### DISCUSSION TITLE
Can I disable image download for visitors?

### DISCUSSION THREAD

**Discussion by PeopleInside** [LiveHelperChat/livehelperchat#1902] [Q&A] [answered]
There is a way to disable the button download for images in widget chat?
When operator send an image this should be not have the download button for the visitor.
An option to disable this should exist.

**Comment by remdex** [comment, marked as answer]
At the moment not

**Reply by PeopleInside** [reply]
Ok thanks, added a request https://github.com/LiveHelperChat/livehelperchat/issues/1903
